env.sh

```diff
@@ -3,84 +3,83 @@
 # Environments' scripts.
 
 # OS
-if [[ $(uname) == 'Darwin' ]]; then
-    SYSTEM_NET="bridge"
-    SOURCE_DIR=$(dirname "$(realpath "$0")")
+if [[ "$(uname)" == 'Darwin' ]]; then
```

Comment:
1. [Test, \[ and \[\[][1]: `[[ ... ]]`
2. [Command Substitution][2]: `"$(command)"`
3. [Single Quoting][3]: `'Darwin'`

[1]: https://google.github.io/styleguide/shell.xml?showone=Quoting#Test,_%5B_and_%5B%5B
[2]: https://google.github.io/styleguide/shell.xml?showone=Command_Substitution#Command_Substitution
[3]: https://google.github.io/styleguide/shell.xml?showone=Quoting#Quoting